v2.1/install-cockroachdb-mac.html

```diff
@@ -0,0 +1,219 @@
+---
+title: Install CockroachDB
+summary: Install CockroachDB on Mac, Linux, or Windows. Sign up for product release notes.
+tags: download, binary, homebrew
+toc: true
+key: install-cockroachdb-mac.html
+redirect_from: install-cockroachdb-mac.html
```

Comment:
I think you meant to say `install-cockroachdb.html` here. This creates an infinite redirect. :)

Reply:
Thanks. Fixed.